partners/balancer.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "Balancer",
+  "description": "Balancer is a decentralized automated market maker (AMM) protocol that represents a flexible building block for programmable liquidity.",
+  "imgUrl": "https://raw.githubusercontent.com/balancer/brand-assets/main/logo/circle-container/logo-balancer-black-128x128.svg",
```

Comment:
There should be a project_url as well, which links to the project

Reply:
Should project URL link to balancer in general, or to the filtered pool deposit page where users can find the positions and only the positions involved in the program?

For now I gave a link to the filter view, but it would be nice both present Balancer as a project/whole thing, and give users an easy link to follow in order to find the things they need to do.

Note still need a frontend PR to merge for the filter view to work.

Reply:
The point of the project url is to prevent users from visiting the wrong websites, so a link to Balancer in general works just fine